### ISSUE TITLE
Installation doesnt work

### ISSUE THREAD

**Issue by PixiBixi** [charles-001/dolphie#17]
Hi, I wanted to try Dolphie, but i've an error out of the box on Ubuntu 20.04 / Python 3.8.10 : 

```
root! x:~$ dolphie
Traceback (most recent call last):
  File "/usr/local/bin/dolphie", line 5, in <module>
    from dolphie.app import main
  File "/usr/local/lib/python3.8/dist-packages/dolphie/__init__.py", line 16, in <module>
    from dolphie.Modules.MetricManager import MetricManager
  File "/usr/local/lib/python3.8/dist-packages/dolphie/Modules/MetricManager.py", line 192, in <module>
    class MetricData:
  File "/usr/local/lib/python3.8/dist-packages/dolphie/Modules/MetricManager.py", line 194, in MetricData
    color: tuple[int, int, int]
TypeError: 'type' object is not subscriptable
```

Freshly installed w/ pip

Regards,

**Comment by charles-001** [comment]
Hi @PixiBixi - thanks for reaching out! This seems to be a Python version issue... can chance you can try Python 3.9?

**Comment by PixiBixi** [comment]
It works on my mac w/ Python 3.11.3, thanks :) 

Kinda sad that it doesn't work w/ built-in Ubuntu 20.04 Python version (Even if its old, its an LTS and supported until 2025 :) )

**Comment by charles-001** [comment]
I wasn't aware typing like `tuple[int, int, int]` wasn't supported in 3.8. I can look to not do that typing so it is supported in 3.8. It's not really important anyway heh

**Comment by charles-001** [comment]
If you stick around after trying it out, look for 3.8 support in the next release :)

**Comment by cheekybuddha** [comment]
Came here to report the same thing on Ubuntu 20.04. Look forward to the release!

**Comment by charles-001** [comment]
@PixiBixi @cheekybuddha - this was fixed in v3.0.5 :)
